packages/wonder-blocks-pill/package.json

```diff
@@ -17,11 +17,12 @@
     "access": "public"
   },
   "dependencies": {
-    "@khanacademy/wonder-blocks-clickable": "^6.0.0",
-    "@khanacademy/wonder-blocks-core": "^12.0.0",
-    "@khanacademy/wonder-blocks-link": "^7.0.8",
-    "@khanacademy/wonder-blocks-tokens": "^4.1.0",
-    "@khanacademy/wonder-blocks-typography": "^3.0.6"
+    "@babel/runtime": "^7.24.5",
```

Comment:
Referencing our thread in Slack to close the loop! https://khanacademy.slack.com/archives/C4PE1QM5Y/p1738795295547179?thread_ts=1738792765.903199&cid=C4PE1QM5Y 

Because we add this dependency, the bundle size for the pill package has changed!